frontend/src/components/misc/ErrorBoundary.tsx

```diff
@@ -219,6 +217,29 @@ function getStringFromInfo(info: InfoItem) {
     }
 }
 
+const CopyToClipboardButton: FC<{message: string, disabled: boolean; isLoading: boolean}> = ({message, disabled, isLoading}) => {
```

Comment:
[question] Why can't you just use `CopyButton`? Is it because it doesn't trigger a toast? We can add a UI library variant that would do that, to keep the logic in the library

https://redpanda-ui.netlify.app/?path=/story/components-copybutton--default

Reply:
We can have a look at this, this was there from before but I think it's unrelated to Modals PR. It was here because the branch was based on top of toasts. I'll add it on my TODO list.